src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier extends ColorSupplier {
+    public static final int MAX_FIGURE = 3;
+    public static final int MAX_SIDE = 20;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(MAX_FIGURE);
+        int firstNumber = random.nextInt(MAX_SIDE);
+        int secondNumber = random.nextInt(MAX_SIDE);
+        int thirdNumber = random.nextInt(MAX_SIDE);
+        String color = String.valueOf(colorSupplier.getRandomColor());
```

Comment:
Use `name()` for getting String representation of enum constant.
```suggestion
        String color = colorSupplier.getRandomColor().name();
```